securedrop/db.py

```diff
@@ -269,17 +274,20 @@ def _scrypt_hash(self, password, salt, params=None):
 
     MAX_PASSWORD_LEN = 128
 
-    def set_password(self, password):
+    @staticmethod
+    def check_password_acceptable(password):
         # Enforce a reasonable maximum length for passwords to avoid DoS
-        if len(password) > self.MAX_PASSWORD_LEN:
+        if len(password) > Journalist.MAX_PASSWORD_LEN:
```

Comment:
I think this does need to be static so it can be preemptively used without creating a Journalist instance. As in, we check the password N times, then make one instance and set it.

Or is that not a good approach?